docs/user_guide/plugin_development/plugin_compatibility_settings.rst

```diff
@@ -0,0 +1,71 @@
+Plugin compatibility settings
+=============================
+
+
+Python compatibility
+--------------------
+
+Your plugin might not be compatible with Python 2 or Python 3.
+In order to prevent users to install them under an incompatible environment,
+you can add a **Python** section to your plug file:
+
+.. code-block:: ini
+
+    [Core]
+    Name = MyPlugin
+    Module = myplugin
+
+    [Documentation]
+    Description = my plugin
+
+    [Python]
+    Version=2+
+
+The possible choices for **Version** are:
+
+- 2 : only compatible with Python 2
+- 3 : only compatible with Python 3
+- 2+: compatible with both
+
+Of there is no Python section, your plugin will be restricted to Python 2 for backward compatibility.
+
+
+Errbot compatibility
+--------------------
+
+Sometimes when your plugin breaks under a specific version of Errbot, you
+might want to warn the user of your plugin and not load it.
+
+
+You can do it by adding an **Errbot** section to your plug file like this:
+
+.. code-block:: ini
+
+    [Core]
+    Name = MyPlugin
+    Module = myplugin
+
+    [Documentation]
+    Description = my plugin
+
+    [Errbot]
```

Comment:
Is this a good section name ('Errbot') for describing plugin version? Why not just [Plugin-Version] or something like that.